tests/null_message.rflx

```diff
@@ -0,0 +1,5 @@
+package Null_Message is
+
+   type Test_Null_Message is null message;
```

Comment:
Please call it `Null_Message` or just `Message`. There is no need for such a bulky name.

Reply:
changed to Message in b010f98